### ISSUE TITLE
Routing: Reduce turns in bicycle routing

### ISSUE THREAD

**Issue by fho** [osmandapp/Osmand#2659]
OSMand seems to calculate the shortest route for bicycling. Often those includes a lot of turns.
This makes the route much slower.
Having to turn often, slows down a cyclist especially on busy roads where you have to change lanes, maybe even cross a traffic light. You often have to reduce your speed already to take the turn.

Straight ways should be preferred, shorter route options that include turns should only be considered if it shortens the way more than X meters

**Comment by Be-ing** [comment]
Have you tried [BRouter](http://brouter.de/brouter/)? It's available in FDroid.

**Comment by fho** [comment]
No, I never heard about Brouter before but it sounds pretty cool.
I'll try it with OSMand

thanks a lot!

**Comment by fho** [comment]
I used brouter for a bicycle tour in Japan (fastbike-asiapacific) and the routing was amazing. :-)

**Comment by Unnamed-1** [comment]
Can we vote on issues please?

Actually, it be good to have an option to be guided like a car except for roads when cars are allowed to drive faster than 50km/h. I.e. most roads in the city area should meet this criteria.
The background is that often I can drive as fast as the cars (since we all are interrupted by the traffic lights) but only as long as there are no pointless interruptions like sudden turns.

But Osmand wants me to leave the optimal road and make a "shortcut" to wait minutes at the traffic light for pedestrians, just to save a couple of meters.

**Comment by fho** [comment]
@Unnamed-1 try brouter like @Be-ing suggested, best bicycle routing ever :-)

Would be best if brouter could become a part osmand

**Comment by matthijskooijman** [comment]
This seems (somewhat) related to #5603.
